connect-base/1.2/docker-entrypoint.sh

```diff
@@ -75,6 +76,15 @@ if [ -z "$CONNECT_PLUGIN_PATH" ]; then
 fi
 echo "Plugins are loaded from $CONNECT_PLUGIN_PATH"
 
+if [[ "${DEBEZIUM_ENABLE_APICURIO}" -eq 1 && ! -z "$EXTERNAL_LIBS_DIR" && -d "$EXTERNAL_LIBS_DIR/apicurio" ]] ; then
```

Comment:
Please use `true` instead of `1`

Reply:
done. I also changed to properly use `$CONNECT_PLUGIN_DIR` now instead of `$KAFKA_CONNECT_PLUGINS_DIR`.